public/pages/HistoricalDetectorResults/components/EmptyHistoricalDetectorResults/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { EmptyHistoricalDetectorResults } from './EmptyHistoricalDetectorResults';
+
+export { EmptyHistoricalDetectorResults };
```

Comment:
Admittedly this is inconsistent within the plugin. The benefits is that it can shorten the import path when used in other components, (only need to specify base folder rather than the explicit file where it is implemented), as well as import multiple components from one directory, instead of one import statement for each. I've seen inconsistencies across plugins, but I prefer it because of the import simplicity.